packages/vuetify/src/components/VCard/VCard.sass

```diff
@@ -111,7 +111,7 @@
 .v-card--flat
   +elevation(0, true)
 
-.v-card--hover
+.v-sheet.v-card--hover
```

Comment:
Technically everything should be prefixed with v-sheet. This could cause issues with any component that extends sheet and a fix should incorporate them as well.